redirect/routes.php

```diff
@@ -262,20 +268,32 @@ class Route {
         ],
     ];
 
-    public function getTargetOfSub($subdomain) {
-        if ($subdomain === 'json') {
+    public function getTargetOfSub($siteType, $redirectUrl=null) {
+
+        if ($siteType === 'json') {
             header('Content-type: application/json');
             die(json_encode($this->routes));
         }
-        if (isset($this->synonyms[ $subdomain ])) {
-            $subdomain = $this->synonyms[ $subdomain ];
+
+        if (isset($this->synonyms[ $siteType ])) {
+            $siteType = $this->synonyms[ $siteType ];
```

Comment:
Please remove those spaces inside the brackets - who does this? :-1: 
yuk!

Reply:
I just adopted the code style there was before, I thought that is what you do in php and the comment you made before was about Strings as keys.